common/src/main/java/com/wynntils/features/user/redirects/ChatRedirectFeature.java

```diff
@@ -40,6 +40,9 @@ public class ChatRedirectFeature extends UserFeature {
     @Config
     public RedirectAction loginAnnouncements = RedirectAction.REDIRECT;
 
+    @Config
+    public RedirectAction mage = RedirectAction.REDIRECT;
```

Comment:
I think you should rename this `mageTeleport` and the `teleport` config to `scrollTeleport`.

Reply:
I did it as just `mage` in case we wanted to extend the config to more mage-specific messages, but I'll change scrollTeleport now.

Reply:
Are you thinking of some specific messages? One clear benefit of having one config per redirect is that the users have full control. Otoh, many configs can be overwhelming, but then again, it can be hard to determine which belong together. Right now I think we have only skill+ability points combined into one config, and horse spawning messages into another. All other are separate.

Reply:
I'll change that over then.